worker/Dockerfile

```diff
@@ -1,23 +1,21 @@
-FROM ubuntu:16.04
+# Get latest Ubuntu LTS.
+FROM ubuntu:latest
```

Comment:
Let's use `22.04` or `20.04` explicitly, would be annoying for it to break later. `latest` is not LTS, it will switch to 22.10 as soon as it is out.
```suggestion
FROM ubuntu:22.04
```

Reply:
Didn't read that far. Still, I think for long-term reliability it would be beneficial to specify the version explicitly.

Reply:
yes, makes sense.